dev/ci/gitlab.bat

```diff
@@ -28,7 +28,18 @@ if exist %DESTCOQ%\ rd /s /q %DESTCOQ%
 call %CI_PROJECT_DIR%\dev\build\windows\MakeCoq_MinGW.bat -threads=1 ^
   -arch=%ARCH% -installer=Y -coqver=%CI_PROJECT_DIR_CFMT% ^
   -destcyg=%CYGROOT% -destcoq=%DESTCOQ% -cygcache=%CYGCACHE% ^
-  -addon="bignums ltac2 equations" -make=N ^
+  -addon=bignums ^
+  -addon=equations ^
+  -addon=mathcomp ^
+  -addon=ltac2 ^
+  -addon=menhir ^
+  -addon=menhirlib ^
+  -addon=compcert ^
+  -addon=vst ^
+  -addon=coquelicot ^
+  -addon=aactactics ^
+  -addon=quickchick ^
```

Comment:
I will be surprised if this succeeds with the current 2h timeout, although we never now. There currently isn't a way to override the default timeout per build unfortunately (see https://gitlab.com/gitlab-org/gitlab-ce/issues/18183), so if we want to bump it, we'll have to do so for all the builds.

Reply:
Afaik the timeout is 3 hours (at least this is stated at some points).
Let's see - I am currently more fighting with random CI failures ...

Reply:
Indeed, it was bumped to 3h. I don't recall this. It used to be 2h not long ago.